### ISSUE TITLE
WDA should use the 'accessibility id' locator strategy instead of 'name'

### ISSUE THREAD

**Issue by jlipps** [facebook/WebDriverAgent#416]
Appium deprecated the 'name' locator strategy years ago, but it appears it's being supported by WDA, which is confusing some Appium users. According to [the spec](https://github.com/SeleniumHQ/mobile-spec/blob/master/spec-draft.md) we should be using the accessibility id locator strategy.

Internally, it can certainly map to the 'name' property of XCUI elements.

If you guys don't want to make this change, we can also make it in Appium instead, but then it introduces another discrepancy between the way our drivers work.

**Comment by marekcirkos** [comment]
I think we do support `accessibility id` -> https://github.com/facebook/WebDriverAgent/blob/master/WebDriverAgentLib/Commands/FBFindElementCommands.m#L115

Tho you are right that, Inspector e.g will show `name` attribute.

**Comment by marekcirkos** [comment]
@jlipps Can you tell me what endpoint is not handling it?

**Comment by jlipps** [comment]
@marekcirkos it's the other way around: you already do support accessibility id, which is great. I'm saying WDA should _not_ handle the `name` strategy, i.e., it should be removed as a valid locator strategy.

**Comment by marekcirkos** [comment]
@jlipps There is no point in keeping this.
